crates/adapters/src/transport/file.rs

```diff
@@ -161,7 +162,8 @@ impl FileInputReader {
                         while remainder > 0 {
                             let n = splitter.read(&mut file, buffer_size, remainder)?;
                             if n == 0 {
-                                todo!();
+                                error!("inconsistency replaying file data");
```

Comment:
Can we give some indication about what could be causing this (the file has changed? an internal bug in Feldera?)

Reply:
I'm changing it to `"{path}: file truncated since originally read"`